.travis.yml

```diff
@@ -138,7 +140,21 @@ script:
   # Show versions
   - python -c 'import statsmodels.api as sm; sm.show_versions();'
   # docbuild and exit, if required
-  - if [ ${DOCBUILD} = true ]; then cd ${SRCDIR}/docs; source ${SRCDIR}/tools/ci/docbuild.sh; exit 0; fi;
+  - |
+    if [ ${DOCBUILD} = true ]; then
```

Comment:
Should this all go in docbuild.sh along with some echo to explain where the script is?

Reply:
Clever. Yes, let me try that.